### PR TITLE
chore: setup material-ui for the project

### LINKED ISSUE #11: Setup Material UI in the project
Next Step for interested people: Setup material-ui in the project

Reference: https://codesandbox.io/s/github/mui-org/material-ui/tree/master/examples/nextjs-with-typescript

### PR DESCRIPTION
**Pull request by pc-beast**
fixes #11

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Show material UI elements on home page like in the reference

**Comment by pc-beast** [comment]
Sure

**Comment by pc-beast** [comment]
@iamareebjamal please check now

**Comment by iamareebjamal** [comment]
There is a flash of unstyled content when I visit the page. See if you can remove that.
Secondly, setup the styles file of material UI as well

**Comment by pc-beast** [comment]
> There is a flash of unstyled content when I visit the page. See if you can remove that.
Sir, I didn't get which unstyled content? Should I add some styling to the element I added?

**Comment by iamareebjamal** [comment]
Cool. Thanks

**Comment by iamareebjamal** [comment]
Styles Provider is still not setup

**Comment by iamareebjamal** [comment]
Our styles definition is still not injected

**Comment by pc-beast** [comment]
@iamareebjamal  should I do [this](https://github.com/mui-org/material-ui/blob/master/examples/nextjs/src/theme.js) with Home.modules.css?

**Comment by iamareebjamal** [comment]
Yes you have to add that but that's not related to Home.module.css

**Comment by pc-beast** [comment]
Got it. I need to add this with theme provider. Should I use same colors as in this file?

**Comment by iamareebjamal** [comment]
Yes, for now

File: pages/_document.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+export default class MyDocument extends Document {
+  render(): JSX.Element {
+    return (
+      <Html lang="en">
+        <Head>
+          {/* PWA primary color */}
+          <link
+            rel="stylesheet"
+            href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
+          />
```
Comment (iamareebjamal) [inline pages/_document.tsx:19]: We need to load fonts locally

**Comment by iamareebjamal** [comment]
The font URL https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap lists 300, 400, 500, 700, but preload and local font include 100 as well. And there is no local font for 400 but there is a preload for it

**Pull request merged**